cypari2/string_utils.pyx

```diff
@@ -38,17 +48,16 @@ cpdef unicode to_unicode(s):
     >>> s1 = to_unicode(b'hello')
     >>> s2 = to_unicode('hello')
     >>> s3 = to_unicode(u'hello')
-    >>> import sys
-    >>> u_type = (unicode if sys.version_info.major <= 2 else str)
-    >>> type(s1) == type(s2) == type(s3) == u_type
+    >>> type(s1) is type(s2) is type(s3) is type(u"")
     True
     >>> s1 == s2 == s3 == u'hello'
     True
+
+    >>> print(to_unicode(1234))
```

Comment:
could add `type(to_unicode(1234)) is type(u"")` similarly to what is in `to_bytes`?